js/components/MilkdropWindow/index.js

```diff
@@ -35,14 +36,17 @@ class PresetsLoader extends React.Component {
   }
 
   async componentDidMount() {
-    const {
-      butterchurn,
-      presetKeys,
-      minimalPresets
-    } = await loadInitialDependencies();
+    const [
```

Comment:
I would just check the URL param here, rather then passing it down.

Reply:
yea, that would be simpler hah